src/app/tests/suites/TV_ApplicationBasicCluster.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Application Basic Cluster Tests
+
+config:
+    cluster: "Application Basic"
+    endpoint: 3
+
+tests:
+    - label: "Change Status Command"
+      command: "ChangeStatus"
+      arguments:
+          values:
+              - name: "status"
+                value: 1
+
+    - label: "Read attribute vendor name"
+      command: "readAttribute"
+      attribute: "vendor name"
+
+      # TODO: Support chars validation
+      # response:
+      #     value: "exampleVendorName1"
```

Comment:
Is this a todo because the strings come through as garbage?

Reply:
Nothing comes out atm. I don't know what happened, but if I understood correctly, logger is not printing out string response due to security concerns.

Reply:
This has nothing to do with logging; the "response" bits are compared to the actual response received from the server.  Please make sure there's an issue tracking figuring out why this is not working; it sounds like something is really broken somewhere....  Might be the "who knows what the string length is" issue.

Reply:
This one is weird. I still don't know what's going on. I've however realised that if I enable TestCluster I can see the values. I can confirm that I've tested other scenarios as well and yes, if TestCluster is enabled I will be able to see text responses otherwise no.